foodsaving/subscriptions/tests/test_fcm.py

```diff
@@ -21,42 +21,60 @@ def logger_warning_mock():
 @contextmanager
 def fcm_reload_without_config():
     from django.conf import settings
-    original = settings.FCM_SERVER_KEY
-    try:
+    if not hasattr(settings, 'FCM_SERVER_KEY'):
+        reload(foodsaving.subscriptions.fcm)
+        yield
+    else:
+        original = settings.FCM_SERVER_KEY
         del settings.FCM_SERVER_KEY
         reload(foodsaving.subscriptions.fcm)
         yield
-    finally:
         settings.FCM_SERVER_KEY = original
         reload(foodsaving.subscriptions.fcm)
 
 
+@contextmanager
+def fake_fcm_key():
```

Comment:
Maybe it can be combined with `fcm_reload_without_config` to be one function that supports bot `with fcm_key(unset=True)` and `with fcm_key(value='foo')`